wikimedia/wikipedia-ios
# Show toast if `SinglePageWebViewController` has no internet connection

**Phabricator:** https://phabricator.wikimedia.org/T323519

### Notes
Show a toast if no internet connection is available when loading pages via `SinglePageWebViewController`.

### Test Steps
1. Load up an article or user talk page
2. After it loads, enable Airplane Mode on your device
3. Tap the More (...) button, and select some of the in-app web bound links in the context menu (Page information, What links here, etc.)
4. Confirm the "No internet connection" toast is displayed
